src/test/scala/example/Codecs.scala

```diff
@@ -0,0 +1,10 @@
+package example
+
+import io.bullet.borer.Codec
+import io.bullet.borer.derivation.MapBasedCodecs.deriveAllCodecs
+
+trait Codecs {
```

Comment:
```suggestion
object Codecs {
```

I know it's test code, but still: with trait, each val will be repeated in each class that pulls the trait in :// this is not erroneous, but still unnecessary